models/storyboard.js

```diff
@@ -19,14 +19,22 @@ function createNav(items) {
     const nav = [];
 
     items.forEach((item, id) => {
-        const [mediaType] = Object.keys(item);
-        // fallback for now until every type has a title field to fill in. (or something for nav)
-        const tmpTitle = item[mediaType].title ? item[mediaType].title : `${id} ${mediaType}`;
-        const title = tmpTitle.replace(/<\/?[^>]+(>|$)/g, "");
+      var elKey = Object.keys(item)[0];
```

Comment:
elKey is the same as mediaType below I think

Reply:
Noted and acted upon